cmd/evm/internal/t8ntool/flags.go

```diff
@@ -125,14 +125,6 @@ var (
 		Name:  "seal.clique",
 		Usage: "Seal block with Clique. `stdin` or file name of where to find the Clique sealing data.",
 	}
-	SealEthashFlag = &cli.BoolFlag{
-		Name:  "seal.ethash",
-		Usage: "Seal block with ethash.",
-	}
-	SealEthashDirFlag = &cli.StringFlag{
-		Name:  "seal.ethash.dir",
-		Usage: "Path to ethash DAG. If none exists, a new DAG will be generated.",
-	}
 	SealEthashModeFlag = &cli.StringFlag{
```

Comment:
This flag is also not used, please remove it.

Reply:
Fixed.